### PR TITLE
added tests for human, inwards, outwards, allee modules

### PR DESCRIPTION
**Pull request by jamesmaino**

### CODE REVIEW

**Comment by rafaqz** [comment]
That's awesome. Just failing in human dispersal:
https://travis-ci.org/rafaqz/Dispersal.jl/jobs/467941982#L843

One thing we should do is make all the testset titles a little more descriptive of what we are testing. But it's no biggie.

We should probably do a few more integration tests as well, at least testing that chaining submodels together works as expected.

File: src/human.jl [hunk 1]
```diff
@@ -43,6 +43,7 @@ precalc_human_dispersal(human::A, cellsize, take, human_exponent, dist_exponent)
     h, w = size(human)
     indices = broadcastable_indices(Int32, human)
     dist = (distances(human) .* cellsize) .^ dist_exponent
+    dist[1] = cellsize/6 * (sqrt(2) + log(1 + sqrt(2))) # mean distance from cell centre
```
Comment (rafaqz) [inline src/human.jl:51]: what does this line do?

Comment (jamesmaino) [reply src/human.jl:51]: we dont want distance from i to i to be zero because we get NaNs when we divide.

Comment (rafaqz) [reply src/human.jl:51]: Ok. You should make that 6 a named variable, magic numbers in the code (besides 0, 1 and 2) are confusing!

Comment (jamesmaino) [reply src/human.jl:51]: I had to look up what magic numbers were. oh right. maybe a function would be better? The equation finds the mean distance from the centre of a square to any surrounding point within the square. 

https://math.stackexchange.com/questions/15580/what-is-average-distance-from-center-of-square-to-some-point

**Comment by rafaqz** [comment]
Oh right that 6 is in the standard equation. A well named utility function
is the way to do that

File: src/human.jl (hunk 1 above)
Comment (rafaqz) [reply src/human.jl:51]: Yeah use a function. (Weird that my email reply didn't show up inline here)

**Pull request merged**